### PR TITLE
fix babelrc plugin

### LINKED ISSUE #14433: Unable to launch the app. Receiving "...facebook.react.common.JavascriptException.."
**Steps:** 

1. Install Status
2. Launch the app
3. Observe the result

**Actual result:**  "...facebook.react.common.JavascriptException.."

**Expected result:** no errors

photo_2022-11-23 18 16 55
[logcat (4).log](https://github.com/status-im/status-mobile/files/10081951/logcat.4.log)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/14433

status: ready

### CODE REVIEW

**Comment by J-Son89** [comment]
Hey Parvesh, I intentionally deleted the babelrc file as it's now using a`babel.config.js` file instead. This has the same configuration so just need to figure out why the app is not reading it

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (J-Son89) [inline .babelrc:1]: Hey Parvesh, I intentionally deleted the babelrc file as it's now using ababel.config.js file instead. This has the same configuration so just need to figure out why the app is not reading it

Comment (Parveshdhull) [reply .babelrc:1]: yes, we can look into that later, why babel.config.js is not working. but let's unblock the develop for now

**Review by J-Son89 — APPROVED** [review]
we can unblock for now and I'll try find the solution so we can delete this again :)

**Comment by churik** [comment]
e2e failures are not related to PR, will be logged/resolved separately, PR can be merged, thank you!

**Pull request merged**